base/uk.ac.stfc.isis.ibex.ui.scriptgenerator/META-INF/MANIFEST.MF

```diff
@@ -22,3 +22,4 @@ Require-Bundle: uk.ac.stfc.isis.ibex.scriptgenerator,
 Automatic-Module-Name: uk.ac.stfc.isis.ibex.ui.scriptgenerator
 Import-Package: uk.ac.stfc.isis.ibex.ui.widgets
 Bundle-ActivationPolicy: lazy
+Export-Package: uk.ac.stfc.isis.ibex.ui.scriptgenerator.dialogs
```

Comment:
Once we've merged #5118 can we make sure the two dialogs are in the same package